helpers/campaignLimiting.js

```diff
@@ -0,0 +1,20 @@
+const BN = require('bn.js')
+
+function shouldWeLimitCampaigns (filteredCampaigns, limit) {
```

Comment:
rename this from `filteredCampaigns` to `campaigns` - since they are not filtered here